jupyterlab/labapp.py

```diff
@@ -580,6 +580,14 @@ class LabApp(NotebookConfigShimMixin, LabServerApp):
         document will be kept in memory forever.""",
     )
 
+    collaborative_document_save_delay = Int(
+        1,
+        allow_none=True,
+        config=True,
+        help="""The second of inactivity to prevent too frequent saving (relevant only in collaborative mode). Defaults to 1s, if None then the
```

Comment:
For the `None` case, the implementation does not look like it is behaving as described in this help:

https://github.com/jupyter-server/jupyter_server_ydoc/blob/9fb989e5fba5a2144a9ccff0f6ea1694a284d1e4/jupyter_server_ydoc/ydoc.py#L260-L262